projects/Mallard/src/screens/issue-screen.tsx

```diff
@@ -386,7 +392,14 @@ const IssueScreenWithPath = React.memo(
                                 retry()
                             }}
                         />
-                        <IssueScreenHeader issue={issue} />
+                        <IssueScreenHeader
+                            issue={issue}
+                            headerStyles={
+                                specialEditionProps
+                                    ? specialEditionProps.headerStyle
```

Comment:
Isnt it easier to do `selectedEdition.headerStyle || underfined`?

Reply:
we can't as we don't know at that point whether `selectedEdition` is a regional or special edition